.chloggen/logiraptor_fix-decision-timer-metric.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
On the fence, do we consider this breaking?

Reply:
After thinking more, it does feel like a breaking change. Updated.